site/src/components/mdx/h3/Heading3.tsx

```diff
@@ -1,10 +1,10 @@
-import { FC, ReactNode } from "react";
+import { ComponentPropsWithoutRef, FC, ReactNode } from "react";
 import styles from "./Heading3.module.css";
 
-type Heading3Props = {
+interface Heading3Props extends ComponentPropsWithoutRef<"h3"> {
   children: ReactNode;
   id?: string;
-};
+}
 
 export const Heading3: FC<Heading3Props> = ({ children, ...rest }) => (
```

Comment:
These are salt dependencies... be mindful at some stage we want to make this a general purpose component, so it might not be part of this repo.